app/controllers/api/v2/environments_controller.rb

```diff
@@ -61,6 +61,13 @@ def destroy
       def allowed_nested_id
         %w(puppetclass_id location_id organization_id)
       end
+
+      def find_resource
```

Comment:
We don't normally override this, since friendly_id can handle it.  Do you know why you removed friendly_id in 8387eb7?